packages/node_modules/@ciscospark/react-component-activity-item/src/index.js

```diff
@@ -46,9 +46,21 @@ export default function ActivityItem(props) {
 /* eslint-disable react/no-unused-prop-types */
 /* eslint-disable-reason passing props via destructuring */
 ActivityItem.propTypes = {
-  activity: PropTypes.object,
+  activity: PropTypes.shape({
+    content: PropTypes.string,
+    displayName: PropTypes.string,
+    files: PropTypes.shape({
+      items: PropTypes.array
+    })
+  }),
   avatarUrl: PropTypes.string,
-  callData: PropTypes.object,
+  callData: PropTypes.shape({
+    callInfo: PropTypes.shape({
+      duration: PropTypes.string,
+      name: PropTypes.string
+    }),
+    status: PropTypes.string
+  }),
```

Comment:
Yea, I left it at its own level here because it needed to be split from activity at some point since the system message doesn't support an activity object. If it gets to where it is too confusing, we can merge them back in and split it out of the activity object further down the chain.